spring-integration-redis/src/test/java/org/springframework/integration/redis/store/RedisChannelMessageStoreTests.java

```diff
@@ -201,4 +204,30 @@ void testJsonSerialization() {
 		assertThat(messages.get(0).getHeaders()).containsKeys(MessageHistory.HEADER_NAME);
 	}
 
+	@Test
+	void testDelayJsonSerialization() {
```

Comment:
Looks like this test duplicates exactly what you are doing in the `RedisMessageGroupStoreTests`.